.github/workflows/windows.yml

```diff
@@ -13,12 +13,12 @@ jobs:
 
    tests:
       name: Test
-      runs-on: windows-2019
+      runs-on: windows-latest
```

Comment:
in my mind it's preferable to pin a specific version of the machine image, to make the CI jobs deterministic. Using the "latest" the CI jobs can start failing without reflecting a change to the code itself, just that the machine image was updated. Is there a good reason to use "latest" instead of the specific version that currently is the latest one?

Reply:
I was being conservative with this commit (see commit message).

>Is there a good reason to use "latest" instead of the specific version that currently is the latest one?

Less maintenance in case when the OS is announced obsolete from github actions.
And another viewpoint is: for those using `*-latest`, it shows the OS is not a requirement (nor related) for the CI jobs; and for those pinned to a specific version, OS version is tied to the jobs/steps in some way.

Reply:
FWIW: I noticed that currently using `macos-latest` yields a warning because it will be changed to `macos-11` soon. And presumably you'd also get a warning if you use an os that would be removed soon. I guess the key is to stay on top of warnings in either case